server/sonar-server/src/main/java/org/sonar/server/measure/index/ProjectMeasuresIndex.java

```diff
@@ -296,7 +295,7 @@ private static QueryBuilder toValueQuery(MetricCriterion criterion) {
 
   @FunctionalInterface
   private interface FacetSetter {
-    void addFacet(SearchRequestBuilder esSearch, Map<String, QueryBuilder> filters);
+    void addFacet(SearchRequestBuilder esSearch, StickyFacetBuilder facetBuilder, ProjectMeasuresQuery query);
```

Comment:
minor: I'd put `facetBuilder` either first or last because it acts as an accumulator